app/Http/Controllers/Admin/AircraftController.php

```diff
@@ -327,10 +322,11 @@ public function expenses(int $id, Request $request): View
         }
 
         if ($request->isMethod('post')) {
-            $expense = new Expense($request->post());
-            $expense->ref_model = Aircraft::class;
-            $expense->ref_model_id = $aircraft->id;
-            $expense->save();
+            $this->financeSvc->addExpense(
+                $request->post(),
+                $aircraft,
+                $aircraft->subfleet->airline_id
```

Comment:
`$aircraft->airline->id` is also possible.

https://github.com/nabeelio/phpvms/blob/ca565fc563d75c673f76207913473c080bceadf0/app/Models/Aircraft.php#L172